js/pages/incidence-rates/components/iranalysis/components/results.html

```diff
@@ -1,66 +1,105 @@
+			</th>
+			<th data-bind="css: classes('tbl-th', 'started', 'smallHeading dataCol')">
+				Started
+			</th>
+			<th data-bind="css: classes('tbl-th', 'duration', 'smallHeading dataCol')">
+				Duration
+			</th>
+			<th data-bind="css: classes('tbl-th', 'view-results')"></th>
+		</thead>
+		<tbody data-bind="foreach: generatedSources">
+			<tr>
+				<td data-bind="css: $component.classes('tbl-col')" nowrap>
+					<!-- ko if: info().executionInfo.status == "COMPLETE" -->
+						<select-sources-btn params="sources: () => [source], callback: $component.runGenerations, wasGenerated: true, label: 'Rerun'"></select-sources-btn>
+					<!-- /ko -->
+				</td>
+				<td data-bind="css: $component.classes('tbl-col'), text: source.sourceName"></td>
+				<!-- ko if: info().executionInfo.status == "COMPLETE" -->
+					<!-- ko if: info().executionInfo.isValid -->
+						<!-- ko with: $component.getSummaryData(info().summaryList) -->
+						<td data-bind="css: $component.classes({ element: 'tbl-col', extra: 'dataCol' })">
+							<div data-bind="text: totalPersons"></div>
+						</td>
+						<td data-bind="css: $component.classes({ element: 'tbl-col', extra: 'dataCol' }), text: cases"></td>
+						<td data-bind="css: $component.classes({ element: 'tbl-col', extra: 'dataCol' }), text: proportion"></td>
+						<td data-bind="css: $component.classes({ element: 'tbl-col', extra: 'dataCol' }), text: timeAtRisk"></td>
+						<td data-bind="css: $component.classes({ element: 'tbl-col', extra: 'dataCol' }), text: rate"></td>
+						<!-- /ko -->
+					<!-- /ko -->
+					<!-- ko ifnot: info().executionInfo.isValid -->
+					<td data-bind="css: $component.classes('tbl-col')" colspan="4"><div class="overflow-ellipsis" style="width:345px" data-bind="text: 'Error: ' + info().executionInfo.message, attr:{title: info().executionInfo.message}"></div></td>
 					<!-- /ko -->
+					<td data-bind="css: $component.classes('tbl-col', 'exec-start', 'dataCol'), text: $component.formatDateTime(info().executionInfo.startTime)"></td>
+					<td data-bind="css: $component.classes({ element: 'tbl-col', extra: 'dataCol' }), text: $component.msToTime(info().executionInfo.executionDuration)"></td>
+					<td align="center" data-bind="css: $component.classes('tbl-col', 'view-reports'), click:$parent.selectSource">
+						<button class="btn btn-sm btn-primary" data-bind="visible: info().executionInfo.isValid">
+							<i class="fa fa-eye"></i> Reports
+						</button>
+						<button class="btn btn-sm btn-danger removeResult">
+                            <i class="fa fa-trash"></i>
+                        </button>
+					</td>
 				<!-- /ko -->
-				<!-- ko ifnot: info().executionInfo.isValid -->
-				<td colspan="4"><div class="overflow-ellipsis" style="width:345px" data-bind="text: 'Error: ' + info().executionInfo.message, attr:{title: info().executionInfo.message}"></div></td>
```

Comment:
We removed the columns that store the execution error message, is this no longer being displayed in this revision?